NickvisionTubeConverter.Shared/Helpers/DependencyManager.cs

```diff
@@ -42,6 +42,33 @@ public static string Ffmpeg
             return "";
         }
     }
+    /// <summary>
+    /// The path for aria2
+    /// </summary>
+    public static string Aria2
+    {
+        get
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return $"{Configuration.ConfigDir}{Path.DirectorySeparatorChar}aria2{Path.DirectorySeparatorChar}aria2.exe";
+            }
+            var prefixes = new List<string>() {
+                Directory.GetParent(Directory.GetParent(Path.GetFullPath(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!))!.FullName)!.FullName,
+                Directory.GetParent(Path.GetFullPath(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!))!.FullName,
+                "/usr"
+            };
+            foreach (var prefix in prefixes)
+            {
+                var path = $"{prefix}/bin/aria2";
```

Comment:
First, idk about Windows but on Linux the binary is "aria2**c**". Second, full path is not needed, yt-dlp will find the binary in $PATH.